requirements.txt

```diff
@@ -10,7 +10,7 @@ python-hglib==2.6.1
 shap==0.29.3
 pandas==0.25.0
 rs_parsepatch==0.2.2
-tqdm==4.33.0
+tqdm~=4.32.2
```

Comment:
Is this change necessary?

Reply:
I guess not, so I've removed it. Feel free to submit another PR if that's needed for some reason.